src/utils/apiAuth.js

```diff
@@ -1,13 +1,18 @@
 import cookie from 'cookie'
 import axios from 'axios'
+import logger from 'loglevel'
 import { StatusCodes } from 'http-status-codes'
 import { isAuthorised } from '@/utils/googleAuth'
 import { paramsSerializer } from '@/utils/urls'
 
+// Sentry doesn't load the config for API routes automatically
+import { Sentry } from '@/root/sentry.server.config'
+
```

Comment:
The suggested `withSentry` helper doesn't load the config so anything hitting the api routes wasn't getting logged unless the same process gets initialised by a standard page request. Since Lambdas don't run continuously there was a danger of missing transactions so I've manually required the server config here (which is why Sentry is exported from the file). The Sentry SDK has protection against `init` being called multiple times so there's no issue if a different request has already called it.